src/spec/dsl.cr

```diff
@@ -64,6 +65,28 @@ module Spec
     exit
   end
 
+  # :nodoc:
+  class_getter randomizer_seed : UInt64?
+  class_getter randomizer : Random::PCG32?
+
+  # :nodoc:
+  def self.order=(mode)
+    seed =
+      case mode
+      when "default"
+        nil
+      when "random"
+        Random::Secure.rand(1..99999).to_u64 # 5 digits or less for simplicity
```

Comment:
Why this arbitrary range? Can't you just use `new_seed`?

Reply:
Or is the seed being printed somewhere? That would be nice.

Reply:
It needs to be printed, as you need to be able to set the seed to repeat a failing spec.

Reply:
Yes the seed must be printed to be able to reproduce a randomly failing run —thought with parallelism involved, it will never be exactly the same run. Hence the simple seed.